xnmt/__init__.py

```diff
@@ -12,6 +12,7 @@
 import logging
 logger = logging.getLogger('xnmt')
 yaml_logger = logging.getLogger('yaml')
+file_logger = logging.getLogger('xnmt_file')
```

Comment:
I didn't understand why you need another logger for this (does this imply a separate log file for crash logs?). Can't you just use the existing one? I think the best thing would be to have the logging information at the end of the standard log file. It's possible that this is what already happens and I just didn't understand the code though.

Reply:
Yes that's what's happening already. I just needed to catch unhandled exceptions and log them explicitly, and using the normal logger this would have resulted in the exception appearing on the console output twice, so I created another logger that logs only to file (the same file as the standard logger), not to the console.

Reply:
OK, cool. Happy to merge this then.